src/scss/coral-talk-iframe/main.scss

```diff
@@ -65,13 +56,25 @@ a {
 	border-bottom-width: 0;
 }
 
+.coral-tabBarSecondary {
+	border-bottom-color: oColorsGetPaletteColor('teal-50');
+}
+
 
 // community guidelines box
 .coral-guidelines {
 	background-color: oColorsGetPaletteColor('wheat');
 	border-color: oColorsGetPaletteColor('wheat');
 }
 
+.coral-guidelines div div p {
```

Comment:
For what this achieves I wouldn't brother with it. The selector is far to specific that it could easily break